odl/space/npy_tensors.py

```diff
@@ -2266,7 +2288,7 @@ def norm(self, x):
         if self.exponent == 2.0:
             return float(np.sqrt(self.const) * _norm_default(x))
         elif self.exponent == float('inf'):
-            return float(self.const * _pnorm_default(x, self.exponent))
+            return float(_pnorm_default(x, self.exponent))
```

Comment:
I've changed this because there are two reasons for it:

1. The mathematical one: `||x||_{p, w} --> ||x||_{inf, w}` should hold for `p --> inf` (we knew that).
2. Also structure-wise, all unweighted p-norms follow the pattern
   ```
   norm = post_map( reduce( map(x) ) )
   ```
   e.g. 

   - `1 <= p < inf`: `map(x) = abs(x) ** p`, `reduce(x) = sum(x)`, `post_map(x) = x ** (1/p)`
   - `p = inf`: `map(x) = abs(x)`, `reduce(x) = max(x)`, `post_map(x) = x`

   Weighted norms have the structure
   ```
   norm = post_map( reduce( weight * map(x) ) )
   ```
   and here it follows naturally that the weights play no role for `p = inf` since they do not change the `max`. Why treat this case completely differently?